package-overrides/github/components/jquery@2.0.3.json

```diff
@@ -1,5 +0,0 @@
-{
-  "main": "jquery",
-  "files": ["jquery.js"],
-  "format": "global"
-}
```

Comment:
Cool, am going to leave all the changes I made to `components/` for posterity.  My primary motivation for removing them was to just eliminate clutter and avoid confusion.

Is it worth leaving a README in this directory indicating that the `npm` variants should be preferred?